.github/label-actions.yml

```diff
@@ -145,7 +145,7 @@
     A maintainer wants you to find and share a specific log message (`packageFiles with updates`) from your repository or environment.
 
 
-    If you self-host Renovate: make sure you run Renovate with `LOG_LEVEL=debug` to get the debug log messages! Next, open the debug-level logs and search for `packageFiles with updates`. This text marks the start of a structured log message that shows every package file, dependency, and update that Renovate found.
+    If you self-host Renovate: make sure you run Renovate with `RENOVATE_LOG_LEVEL=debug` to get the debug log messages! Next, open the debug-level logs and search for `packageFiles with updates`. This text marks the start of a structured log message that shows every package file, dependency, and update that Renovate found.
```

Comment:
Do we need to add a note that LOG_LEVEL may be needed for older versions? Or just hope there isn't too much confusion during the changeover?

Reply:
True, we can leave the old way, as it's still supported